src/Formatter/DateTimeFormatter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Formatter;
+
+use Symfony\Component\PropertyAccess\PropertyPath;
+
+final class DateTimeFormatter implements DataFormatterInterface
+{
+    /** @var null|string */
+    private $dateTimeFormat = null;
```

Comment:
It's a private property now, can't be used outside the class, so we can name it Oskar if we please